### PR TITLE
Using personal access token to create PR for yarn lockfile update.

### PR DESCRIPTION
**Pull request by dennisoelkers**
## Motivation and Context

This PR is changing the workflows which are creating new PRs to do this with a personal access token, so other workflow are starting for them as well. All of these are started on a schedule, so they cannot produce an infinite loop. [(docs)](https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow)

/nocl

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: .github/workflows/fix-linter-hints.yml [hunk 1]
```diff
@@ -55,3 +55,4 @@ jobs:
           committer: Dr. Lint-a-lot <garybot2@graylog.com>
           commit-message: Running lint --fix
           delete-branch: true
+          token: ${{ secrets.PAT_GRAYLOG_PROJECT_INTERNAL_WORKFLOW_RW }}
```
Comment (mpfz0r) [inline .github/workflows/fix-linter-hints.yml:58]: I think that won't work, the token lacks permissions to create PRs.
I can move that job to another repo if you like.

Comment (bernd) [reply .github/workflows/fix-linter-hints.yml:58]: How about creating a single job in graylog-project-internal that updates the `yarn.lock` files in all repos in a project manifest?

Comment (mpfz0r) [reply .github/workflows/fix-linter-hints.yml:58]: How about this then?

https://github.com/Graylog2/graylog2-server/pull/17657

**Comment by dennisoelkers** [comment]
Closing in favor of #17657.

**Pull request closed without merging**